PowerEditor/src/Parameters.cpp

```diff
@@ -1681,11 +1681,11 @@ bool NppParameters::getUserParametersFromXmlTree()
 std::pair<unsigned char, unsigned char> NppParameters::addUserDefineLangsFromXmlTree(TiXmlDocument *tixmldoc)
 {
 	if (!tixmldoc)
-		return std::pair<unsigned char, unsigned char>(0, 0);
+		return std::pair<unsigned char, unsigned char>('0', '0');

```

Comment:
in C++ `0 != '0'`   This needs to be rewritten as either:

```
return std::pair<unsigned char, unsigned char>((unsigned char)0, (unsigned char)0);
```
or optionally, using `escape character`  syntax for ASCII NUL:
```
return std::pair<unsigned char, unsigned char>('\0', '\0');
```

Reply:
Nice catch. 👍